src/scenes/home/families/jumboQuote/jumboQuote.css

```diff
@@ -1,15 +1,26 @@
-
 .quotes {
   color: #445366;
+  max-width: 1100px;
```

Comment:
Between these styles and the styles I wrote for `shared/quoteBanner` we've implemented styles for quotes twice. I think it may be useful to extract some mix of our styles into some kind of shared Quote component where we can pass props such as `isHeader` and `hasLines`. 

You can see the `<QuoteBanner>` component in action on `/team`